### DISCUSSION TITLE
Support oraxen blocks for missions

### DISCUSSION THREAD

**Discussion by AxDevv** [BG-Software-LLC/SuperiorSkyblock2#1364] [Pending Feature Requests] [answered]
Hello, I would like to propose this suggestion which would be interesting to create missions with custom blocks via Oraxen

**For example:**

required-blocks:
  '1':
    types:
      - 'ORAXEN:PYRITE_ORE'
    amount: 5
    
icons:
  not-completed:
    type: BOOK
    customModel: 7
    name: '&aQuest 8'
    lore:
      - ''
      - '&6Objectives:'
      - '&8 - &7Break &f{value_oraxen:pyrite_ore}/5&7 Pyrite ore'

**Comment by Lemvs** [comment, marked as answer]
Implemented for a long time.

@OmerBenGera you can mark this answered and close it.
